programs/survival/fakeplayer_inv_editor.sc

```diff
@@ -0,0 +1,107 @@
+__config()->{
+    'scope'->'global',
+    'stay_loaded'->true
+};
+create_datapack('invupd', 
+    {
+        'readme.txt' -> ['this data pack is created by scarpet','please dont touch it'],
+        'data' -> {
+            'chyx/advancements/xd.json'-> {
+                'rewards' -> {'function' -> 'chyx:invupd'},
+                'criteria' -> {
+                    'example' -> {
+                        'trigger' -> 'minecraft:inventory_changed',
+                        'conditions' -> {}
+                    }
+                }
+            },
+            'chyx/functions/invupd.mcfunction' -> 'script run signal_event(\'invupd\', null, player())\nadvancement revoke @s only chyx:xd'
+        }
+    }
+);
+global_nope=nbt('{nope:nopeChYx'+rand(1)+'nope}');
+
+global_slotmap=[[-1,7],[-2,1],[-3,2],[-4,3],[-5,4],...map(range(9),[_,45+_]),...map(range(27),[9+_,18+_])];
+
+
+global_fakeplayersscreen={};
+
+__on_player_interacts_with_entity(creativeplayer, fakeplayer, hand)->(
+    if(
+        hand != 'mainhand', return(),
+        fakeplayer~'player_type' != 'fake', return(),
+        creativeplayer~'player_type' == 'fake', return(),
+        //creativeplayer~'gamemode' != 'creative', return(),
+        global_fakeplayersscreen:fakeplayer, return()
+    );
+    
+
+    screen=create_screen(creativeplayer,'generic_9x6',fakeplayer~'name',_(screen, player, action,data,outer(fakeplayer))->(
+        
+        if(action=='close',(
+                    //screentoplayer(fakeplayer,screen);
+                    drop_item(screen,-1);
+                    //close_screen(screen);//end_portal/die
+                    return()
+        ));
+        if(action=='slot_update'
+        &&0<=data:'slot'
+        &&data:'slot'<54
+        ,screentoplayer(fakeplayer,screen);return());
+        if(data:'slot'==null,return());
+        if(inventory_get(screen, data:'slot'):2==global_nope,return('cancel'));
+        
+        
+    ));
+
+    global_fakeplayersscreen:fakeplayer=screen;
+
+    loop(54,inventory_set(screen,_, 1, 'minecraft:structure_void',global_nope));
+    //inventory_set(screen,fakeplayer~'selected_slot'+9, 1, 'minecraft:barrier',global_nope);
```

Comment:
Why is this line commented out?

Reply:
that part of code need a pr to carpet mod.

seems that it is not to be merged, i comment it out.